content/javascript/concepts/jquery/terms/click/click.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: '.click()'
+Description: 'Attaches an event handler to the "click" event of an HTML element.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'DOM'
+  - 'Events'
+  - 'Functions'
+  - 'jQuery'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **`.click()`** method attaches an [event handler](https://www.codecademy.com/resources/docs/javascript/events) to the `"click"` event of an [HTML element](https://www.codecademy.com/resources/docs/html/elements).
```

Comment:
Not sure why the second click in this sentence has code highlighting and quotes, seems like it should be one or the other

Reply:
This is because the first instance is referring to a _method_ whereas the other one refers to the name of a specific _event_ (which is usually in quotes), hence the difference in highlighting.